### DISCUSSION TITLE
Creating Events--> Sends Messages --> Causes Spam Triggering

### DISCUSSION THREAD

**Discussion by dadams39** [microsoftgraph/msgraph-sdk-go#265] [Q&A] [answered]
Related to: 
- https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/ability-to-update-or-create-an-event-without-sending-a/idi-p/2335070
- https://docs.microsoft.com/en-us/answers/questions/319147/calendar-event-update-no-email-notifications.html

There should be a way to create/update events that doesn't cause messages to be sent out. Like `send-ntofications := false`.

Failure to have this feature causes messages like
```
Your message wasn't delivered because the recipient's email provider rejected it.
Diagnostic information for administrators:

Generating server: BN7PR17MB2258.namprd17.prod.outlook.com

foobar@someLocation.onmicrosoft.com
Remote Server returned '550 5.7.708 Service unavailable. Access denied. traffic not accepted from this IP. For more information please go to http://go.microsoft.com/fwlink/?LinkId=526653 AS(7230) [PH0PR17MB4519.namprd17.prod.outlook.com]'
```

This option has been requested several times on the forums and I'm not sure how to elevate the status. Additionally, the process for getting an account off the `SPAM` list seems unnecessarily complicated.

**Comment by baywet** [comment, marked as answer]
Hi @dadams39,
This repository is dedicated to the Go SDK itself and doesn't get attention from the different service teams behind the APIs.
The best thing to do for new features requests is to [post on the feedback portal](https://aka.ms/graphfeedback), where existing requests will get their statuses updated.

As per the spam issue, this might be caused by using a developer tenant, I've noticed similar behaviour with my demo/dev tenants. The best thing to do is get in touch with the admin of the receiving tenant to investigate and/or Office 365 support.

**Reply by baywet** [reply]
https://feedbackportal.microsoft.com/feedback/idea/c96a3e47-42d2-ec11-a7b6-000d3a54e061

**Reply by dadams39** [reply]
@baywet are you sure there is not a `SingleValueLegacyExtendedProperty` that would solve this?

**Reply by baywet** [reply]
maybe but that's a question for the API owners, not the SDK team. If you get additional information, we're not aware of, please cross post it here for others.
